### ISSUE TITLE
Android demo running error

### ISSUE THREAD

**Issue by unparalleled-ysj** [TensorSpeech/TensorFlowTTS#338]
Hi, When I tried to replace the ljspeech tflite model with a new tflite model, I encountered this problem: 
`com.example.myapplication E/InputWorker: Exception: 
    java.lang.IllegalArgumentException: Cannot convert between a TensorFlowLite tensor with type FLOAT32 and a Java object of type [I (which is compatible with the TensorFlowLite type INT32).
        at org.tensorflow.lite.Tensor.throwIfTypeIsIncompatible(Tensor.java:427)
        at org.tensorflow.lite.Tensor.getInputShapeIfDifferent(Tensor.java:287)
        at org.tensorflow.lite.NativeInterpreterWrapper.run(NativeInterpreterWrapper.java:146)
        at org.tensorflow.lite.Interpreter.runForMultipleInputsOutputs(Interpreter.java:360)
        at com.example.myapplication.module.FastSpeech2.getMelSpectrogram(FastSpeech2.java:67)
        at com.example.myapplication.tts.InputWorker$InputText_xsp.proceed(InputWorker.java:142)
        at com.example.myapplication.tts.InputWorker$InputText_xsp.access$700(InputWorker.java:95)
        at com.example.myapplication.tts.InputWorker.lambda$new$0$InputWorker(InputWorker.java:52)
        at com.example.myapplication.tts.-$$Lambda$InputWorker$bb1bzD20SufeNF2Y90dc4o1Q4iE.run(Unknown Source:2)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:462)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)`

Is there a problem when I converted to the tflite model, hope to get your help ! @mapledxf @dathudeptrai

**Comment by unparalleled-ysj** [comment]
I found the problem. The model used in the Android demo is an old version, which also contains the attention mask that needs to be removed at the input.
